setup.cfg

```
@@ -1,6 +1,6 @@
 [metadata]
 name = qiskit-qir
```

Should the name be changed to `iqm-qiskit-qir`?

```suggestion
name = iqm-qiskit-qir
```